app/celery/process_comp_and_pen.py

```diff
@@ -0,0 +1,119 @@
+from dataclasses import dataclass
+from uuid import uuid4
+
+from flask import current_app
+from sqlalchemy.orm.exc import NoResultFound
+
+from notifications_utils.statsd_decorators import statsd
+
+from app import notify_celery
+from app.dao.service_sms_sender_dao import dao_get_service_sms_sender_by_id
+from app.models import (
+    Service,
+    Template,
+)
+from app.notifications.send_notifications import lookup_notification_sms_setup_data, send_notification_bypass_route
+from app.va.identifier import IdentifierType
+
+
+@dataclass
+class DynamoRecord:
+    participant_id: str
+    payment_amount: str
+    vaprofile_id: str
+
+
+@notify_celery.task(name='comp-and-pen-batch-process')
+@statsd(namespace='tasks')
+def comp_and_pen_batch_process(records: list[dict[str, str]]) -> None:
```

Comment:
Did we want to leave a comment about this being triggered by a Glue Script?

Reply:
We don't do that with other code. I don't believe that's necessary here. I do need to update some of our other docs though.